account-kit/core/src/metrics.ts

```diff
@@ -0,0 +1,40 @@
+import { createLogger } from "@account-kit/logging";
+import { VERSION } from "./version.js";
+
+export type ConfigCreatedData = {
+  ssr: boolean;
+  chainIds: number[];
+};
+
+export type CoreEventsSchema = [
+  {
+    EventName: "config_created";
+    EventData: ConfigCreatedData;
+  },
+  {
+    EventName: "signer_authnticate";
+    EventData:
+      | {
+          authType: "email" | "passkey_anon" | "passkey_email";
+        }
+      | { authType: "oauth"; provider: string };
+  },
+  {
+    EventName: "signer_sign_message";
+  },
+  {
+    EventName: "client_send_uo";
+  },
```

Comment:
I don't think we need this here actually if it's being added to the infra package

Reply:
Agreed it'll cause dupes and confusion down stream when we go to measure this. Also like the approach of having a single place to define the event schema.

Reply:
> Also like the approach of having a single place to define the event schema.

Per package right? I went with per package but central location because it helps keep a separation of concerns between packages while still maintaining high discoverability for the metric names

Reply:
Yeah per package haha